Cargo.toml

```diff
@@ -1,34 +1,42 @@
 [package]
-name = "tealdeer"
-version = "0.4.0"
 authors = ["Danilo Bargen <mail@dbrgn.ch>"]
-repository = "https://github.com/dbrgn/tealdeer/"
+description = "Fetch and show tldr help pages for many CLI commands. Full featured offline client with caching support."
 homepage = "https://github.com/dbrgn/tealdeer/"
 license = "MIT/Apache-2.0"
+name = "tealdeer"
 readme = "README.md"
-description = "Fetch and show tldr help pages for many CLI commands. Full featured offline client with caching support."
+repository = "https://github.com/dbrgn/tealdeer/"
+version = "0.4.0"
 
 [[bin]]
 name = "tldr"
 path = "src/main.rs"
 
-[features]
-logging = ["env_logger"]
-dev = ["clippy"]
-
 [dependencies]
-log = "^0.3"
-xdg = "^2.0"
-tar = "^0.4"
-curl = "^0.4"
-time = "^0.1"
-flate2 = "^0.2"
-docopt = "^0.6"
-walkdir = "^1.0"
-ansi_term = "^0.9"
-rustc-serialize = "^0.3"
-clippy = {version = "~0.0.37", optional = true}
-env_logger = { version = "^0.3", optional = true }
+ansi_term = "0.10.2"
+curl = "0.4.8"
+docopt = "0.8.1"
+flate2 = "0.2.20"
+log = "0.3.8"
+serde = "1.0.21"
+serde_derive = "1.0.21"
+serde_json = "1.0.6"
+tar = "0.4.14"
+time = "0.1.38"
+walkdir = "2.0.1"
+xdg = "2.1.0"
+
+[dependencies.clippy]
+optional = true
+version = "0.0.174"
+
+[dependencies.env_logger]
+optional = true
+version = "0.4.3"
```

Comment:
Did you use some kind of auto-formatting tool on this file? Can you change this back to the inline-style? I prefer that :)

Reply:
Ah yes, `cargo upgrade` does not preserve formatting and always outputs in this format as far as I know. I can massage this file into inline shape later today.